logql/v2/parser_test.go

```diff
@@ -1214,28 +1214,6 @@ func TestParseExpr(t *testing.T) {
 				},
 			},
 		},
-		{
```

Comment:
hey @periklis 
I'm removing this test because now it fails, as the enclosing parenthesis are removed (so the parsed expression is not strictly equal to the input expression)
But I would say this test intention is covered via the new test I'm adding in ast_test:

```go
		{
			input:  `(count_over_time({first="value"}[10h]))`,
			output: `count_over_time({first="value", second="next"}[10h])`,
		},
```

what do you think?

Reply:
test restored